foodsaving/conversations/tests/test_api.py

```diff
@@ -211,13 +210,14 @@ def test_enable_email_notifications(self):
 
     def test_send_email_notifications(self):
         users = [VerifiedUserFactory() for _ in range(3)]
-        [self.conversation.join(u) for u in users]
+        [self.group.add_member(u) for u in users]
```

Comment:
Checking the active-ness of the group member also means a by-product is we check they are a member at all, so they can't just be in the conversation as was the case before in the test.